Content.Shared/Throwing/ThrownItemComponent.cs

```diff
@@ -50,5 +50,11 @@ public sealed partial class ThrownItemComponent : Component
         /// </summary>
         [DataField]
         public Vector2? OriginalScale = null;
+
+        /// <summary>
+        ///     If the events on collision will actually be raised
+        /// </summary>
+        [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+        public bool HitEvents = true;
```

Comment:
this name is legit sinful in it's lack of descriptiveness.

Reply:
i like sin